terraform/full_environment/files/analytics_transfer_function/main.py

```diff
@@ -65,8 +65,8 @@ def yesterday():
                         ga.user_pseudo_id AS userPseudoId,
                         FORMAT_TIMESTAMP("%FT%TZ",TIMESTAMP_MICROS(ga.event_timestamp)) AS eventTime,
                         (SELECT value.string_value FROM UNNEST(event_params) WHERE key = 'search_term') AS searchQuery,
-                        safe_cast(regexp_extract((SELECT value.string_value FROM UNNEST(event_params) WHERE key = 'page_location'), "page=(\\d+)" ) as int64)-1 as `offset`,
-                        regexp_extract((SELECT value.string_value FROM UNNEST(event_params) WHERE key = 'page_location'), "order=([a-zA-Z\\-]+)" ) as orderBy,
+                        safe_cast(regexp_extract((SELECT value.string_value FROM UNNEST(event_params) WHERE key = 'page_location'), "page=(\\\\d+)" ) as int64)-1 as `offset`,
+                        regexp_extract((SELECT value.string_value FROM UNNEST(event_params) WHERE key = 'page_location'), "order=([a-zA-Z\\\\-]+)" ) as orderBy,
```

Comment:
Could these two lines be squashed into the previous commit? I think the changes will be easier to follow later if future us don't need to step over missing character commits.